config/jobs/kubernetes/kops/kops-periodics-network-plugins.yaml

```diff
@@ -204,6 +204,38 @@ periodics:
     testgrid-dashboards: google-aws, sig-cluster-lifecycle-kops
     testgrid-tab-name: kops-aws-cni-kopeio
 
+- interval: 2h
+  name: e2e-kops-aws-cni-kuberouter
+  labels:
+    preset-service-account: "true"
+    preset-aws-ssh: "true"
+    preset-aws-credential: "true"
+  decorate: true
+  decoration_config:
+    timeout: 90m
+  spec:
+    containers:
+    - command:
+      - runner.sh
+      - /workspace/scenarios/kubernetes_e2e.py
+      args:
+      - --cluster=e2e-kops-aws-cni-kuberouter.test-cncf-aws.k8s.io
+      - --deployment=kops
+      - --env=KOPS_DEPLOY_LATEST_URL=https://storage.googleapis.com/kubernetes-release/release/latest.txt
+      - --env=KOPS_KUBE_RELEASE_URL=https://storage.googleapis.com/kubernetes-release/release
+      - --extract=release/latest
+      - --ginkgo-parallel
+      - --kops-args=--networking=kube-router
```

Comment:
Let's use the Ubuntu image, same as for the other network plugins:
```suggestion
      - --kops-args=--networking=kube-router --image=099720109477/ubuntu/images/hvm-ssd/ubuntu-bionic-18.04-amd64-server-20200112
```

Reply:
ah good idea. updated 👍